scripts/src/blockchain/txSender.ts

```diff
@@ -14,53 +21,78 @@ interface WhitelistPayloadTemp {
 }
 
 export class TxSender {
-	constructor(private readonly _keypair: KeyPair, private readonly _appId: string, private readonly _server: Server) {
+	private readonly CHANNEL_TOP_UP_TX_COUNT = 1000;
```

Comment:
Please put this param in the config file.

Reply:
done